site2/docs/cookbooks-retention-expiry.md

```diff
@@ -31,7 +31,7 @@ By default, when a Pulsar message arrives at a broker it will be stored until it
 
 Retention policies are particularly useful if you intend to exclusively use the Reader interface. Because the Reader interface does not use acknowledgements, messages will never exist within backlogs. Most realistic Reader-only use cases require that retention be configured.
 
-When you set a size limit of, say, 10 gigabytes, then messages in all topics in the namespace, *even acknowledged messages*, will be retained until the size limit for the topic is reached; if you set a time limit of, say, 1 day, then messages for all topics in the namespace will be retained for 24 hours.
+When you set a size limit of, say, 10 gigabytes, then acknowledged messages in all topics in the namespace will be retained until the size limit for the topic is reached; if you set a time limit of, say, 1 day, then acknowledged messages for all topics in the namespace will be retained for 24 hours. The retention settings do not affect unacknowledged messages -- these are instead controlled by the backlog quota and retention policy.
 
 TODO: Confirm this behavior?
```

Comment:
shall we delete line 36?

Reply:
I don't know if that TODO was referring to the paragraph I changed, or the one after. Seems like the one after.

Reply:
Yes, just delete it.

Reply:
Done

Reply:
Thanks for your efforts, I've already approved it.